components/combo/SharingMenu.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div>
+    <head>
```

Comment:
These head tags should be removed. It’s already handled on the combo detail page.